package.json

```diff
@@ -9,7 +9,7 @@
     "dist/*"
   ],
   "engines": {
-    "node": "^10.14.0"
+    "node": "^12.18.0"
```

Comment:
We have CI set up to test this on Node.js >= 10 <=14 - We should use these ranges, otherwise npm will yell at you with engines 12 when installing it with node 10. Consider removing the engines completelty as we did in configuru here https://github.com/AckeeCZ/configuru/pull/28 .

Reply:
`engines` removed